app/Livewire/Server/Destinations.php

```
@@ -29,8 +30,7 @@ public function mount(string $server_uuid)
 
     private function createNetworkAndAttachToProxy()
     {
-        $connectProxyToDockerNetworks = connectProxyToNetworks($this->server);
-        instant_remote_process($connectProxyToDockerNetworks, $this->server, false);
+        ConnectProxyToNetworksJob::dispatchSync($this->server);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Jobs\ConnectProxyToNetworksJob' in method 'createNetworkAndAttachToProxy'. (undefined)

(StaticAccess)